1-mandatory/3-reading-list.js

```diff
@@ -56,11 +97,21 @@ As an example for this exercise, you might do the following steps
 
 test("books are logged", function () {
   expectLogBooksToLog([
-    "The Hobbit by J.R.R. Tolkien",
-    "The Map of Salt and Stars by Jennifer Zeynab Joukhadar",
-    "Dietland by Sarai Walker",
-    "A Place for Us by Fatima Farheen Mirza",
-    "The House of Impossible Beauties by Joseph Cassara",
+    "Harry Potter by J.K Rowlling",
+    "The Hobbit by J.R.R Tolkien",
+    "Rich Dad Poor Dad by Robert T. Kiyosaki",
+    "Hello World by Hannah Fry",
+    "The age of AI by Henry A Kissinger",
+  ]);
+});
+
+test("books you have already read", function () {
+  expectLogBooksToLog([
+    "You've already read Harry Potter by Mark J.K Rowlling",
```

Comment:
who is Mark J.K Rowlling? 😜 This should just be `J.K Rowlling` as per your array above.